assets/scss/thema/lustrum.scss

```diff
@@ -1,121 +1,75 @@
+$headings-font-family: 'Orpheus Pro', 'Source Sans Pro', Helvetica, Arial, sans-serif;
 
-#forumtabel tr.ongelezenvanaf td {
-  background-color: #c52d30 !important;
-}
+@import '../base';
 
-/*Koppen in rood*/
-
-h1,
-a,
-a:hover,
-.cd-primary-nav > li > a:hover,
-.cd-primary-nav > li > a.selected {
-  color: #c52d30;
+body {
+	background-color: $lustrum-kleur-rood;
 }
 
-/*NAVIGATIE MENU*/
-
-/*Zorg voor rode borders*/
-
-#zijbalk .blok .zijbalk-kopje {
-	border: 0 solid #c52d30;
-	border-bottom-width: 2px;
-	margin-right: 40px;
-  background-image: none;
+#menu {
+	background: $lustrum-kleur-donker-rood;
+	border-bottom-color: $lustrum-kleur-goud;
+	a {
+		color: $lustrum-kleur-goud;
+	}
 }
 
-/* letters moeten wit */
-
 #zijbalk .blok .zijbalk-kopje a {
-  color: #ffffff;
-}
-
-/*Veel tekst van de zijbalk in het wit*/
-
-#zijbalk .ishetal .ja,
-#zijbalk .ishetal .nee,
-.staatrood,
-#zijbalk .blok .item a,
-#zijbalk .ishetal,
-#zijbalk .blok .item {
-  color: #ffffff;
-}
-
-/*Andere koppen ook in het wit*/
-
-.cd-primary-nav .cd-secondary-nav a,
-.cd-primary-nav .cd-secondary-nav > li > a:hover,
-.cd-primary-nav .cd-secondary-nav > li > a {
-  color: #ffffff;
+		font-family: $headings-font-family;
+		color: $lustrum-kleur-licht-goud;
+		text-transform: uppercase;
 }
 
-/*Hovering over navigatiemenuknop zorgt voor onderstreeping*/
-
-.cd-primary-nav .cd-secondary-nav a:hover {
-  text-decoration: underline !important;
-  color: #ffffff;
-}
-
-/*Streepje onder naam*/
-
-.cd-primary-nav > li > a:hover,
-.cd-primary-nav > li > a.selected {
-  box-shadow: inset 0 -2px 0 #ffffff;
-}
-
-/*Zijbalk grens van het scherm in donkerblauw*/
-
 #zijbalk {
-  border-color: #151b29;
+	background-color: $lustrum-kleur-donker-rood;
+	.blok .item {
+		color: $lustrum-kleur-goud;
+		a {
+			color: $lustrum-kleur-goud;
+		}
+	}
 }
 
-/*Logo*/
-
-#zijbalk .cd-beeldmerk {
-  background-image: url("../../images/onontdekt_beeldmerk.png");
+.bb-block {
+	box-shadow: $lustrum-kleur-goud;
 }
 
-/*KNOPPEN*/
-/*Knoppen in blauw gradient*/
-
-.btn,
-input[type=submit],
-input[type=button],
-.progress-bar-info,
-.btn:focus,
-input[type=submit]:focus,
-input[type=button]:focus {
-  background-image: linear-gradient(to bottom, #151b29, #2d7eb2);
+.ishetal:after {
+	content: '';
+	display: block;
+	height: 156px;
+	background: url('../../images/lustrum12/Lustrum12.png') no-repeat center center;
+	background-size: contain;
+	margin: 20px 0;
+	clear: left;
 }
-
-/*Voor een mouse hover wordt de gradient omgedraaid*/
-
-.btn:hover,
-input[type=submit]:hover,
-input[type=button]:hover {
-  background-image: linear-gradient(to bottom, #2d7eb2, #151b29);
+.ishetal {
+	.nee {
+		color: $lustrum-kleur-rood;
+	}
+	.ja {
+		color: $lustrum-kleur-goud;
+	}
+	color: $lustrum-kleur-goud;
 }
 
-/*Actieve knoppen in rood gradient -> zo ontstaat verschil tussen hover en active*/
-
-.btn.active {
-  background-image: linear-gradient(to bottom, #c52d30, #f0262a);
+.lustrum_button {
```

Comment:
Dit is toch dezelfde style als in _zijbalk.scss?

Reply:
De .lustrum_button bedoel je? Ik heb daar de kleuren omgedraaid, dus normaal goud en wordt wit als je overheen gaat. In _zijbalk.scss is dat juist andersom.